lib/internal/js_stream_socket.js

```diff
@@ -157,6 +157,7 @@ class JSStreamSocket extends Socket {
     let pending = bufs.length;
 
     this.stream.cork();
+    // Prefer `var` over `let` for performance optimization
```

Comment:
Small nit:
```suggestion
    // Prefer `var` over `let` for performance optimization.
```

Reply:
thanks @lundibundi. fixed

Reply:
reworded, as build results warned: `Line should be <= 72 columns`